.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Rust CI"
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Build contracts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
```

Comment:
I think there is no need in rust here.

Reply:
there will be (coming up in the next PR.. ;-)